src/components/metadata/metadata.test.js

```diff
@@ -245,4 +246,48 @@ describe('MetaData', () => {
       expect(textOf(rowValue(row))).toEqual(['Default']);
     });
   });
+
+  describe('Plot nodes', () => {
+    it('shows the node type as an icon', () => {
+      const wrapper = mount({ nodeId: bullPlotNodeID });
+      expect(rowIcon(wrapper).hasClass('pipeline-node-icon--icon-plot')).toBe(
+        true
+      );
+    });
+
+    it('shows the node name as the title', () => {
+      const wrapper = mount({ nodeId: bullPlotNodeID });
+      expect(textOf(title(wrapper))).toEqual(['Plot:bull']);
+    });
+
+    it('shows the node type as text', () => {
+      const wrapper = mount({ nodeId: bullPlotNodeID });
+      const row = rowByLabel(wrapper, 'Type:');
+      expect(textOf(rowValue(row))).toEqual(['data']);
+    });
+
+    it('shows the node filepath', () => {
+      const wrapper = mount({ nodeId: bullPlotNodeID });
+      const row = rowByLabel(wrapper, 'File Path:');
+      expect(textOf(rowValue(row))).toEqual(['-']);
+    });
+
+    it('shows the node parameters', () => {
+      const wrapper = mount({ nodeId: bullPlotNodeID });
+      const row = rowByLabel(wrapper, 'Parameters (-):');
+      expect(textOf(rowValue(row))).toEqual([]);
+    });
+
+    it('shows the node tags', () => {
+      const wrapper = mount({ nodeId: bullPlotNodeID });
+      const row = rowByLabel(wrapper, 'Tags:');
+      expect(textOf(rowValue(row))).toEqual(['Small']);
+    });
+
+    it('shows the node pipeline', () => {
+      const wrapper = mount({ nodeId: bullPlotNodeID });
+      const row = rowByLabel(wrapper, 'Pipeline:');
+      expect(textOf(rowValue(row))).toEqual(['Default']);
+    });
```

Comment:
Other than the above generic tests to test the data, please add in new tests to test the existence of the plot in the metadata panel, such as testing if certain classes related to the plot exists, etc.